shared/src/environment.ts

```diff
@@ -0,0 +1,17 @@
+export type EnvironmentData = {
+  waste?: WasteData;
+};
+
+export type WasteData = {
+  questionnaireData?: WasteQuestionnaireData;
+  submitted?: boolean;
```

Comment:
Is there value to the undefined state here as opposed to just being false if you haven't submitted yet?

Reply:
Given that `submitted` doesn't get updated till the user clicks save, I don't think there is a reason for it to be undefined. I will make the change!